pkg/config/env/cloud_runtime.go

```diff
@@ -0,0 +1,11 @@
+package env
+
+import (
+	"github.com/gobuffalo/envy"
+)
+
+// CloudRuntime returns the Cloud Provider
+// underneath which the Proxy/Registry is running.
+func CloudRuntime() string {
```

Comment:
@michalpristas It's a bit different because the Go Env can be "development", "staging", or "production". Meaning, you can potentially be on GCP but still running as "staging". CloudRuntime is just to say what cloud are you running on. "development" is overlapping from both because you can be running on your local machine. But that name can be changed to "local" or "none" if you think is better.

Reply:
@robjloranger updated to "none"